op-challenger/game/keccak/matrix/matrix_test.go

```diff
@@ -55,32 +56,6 @@ type testData struct {
 	Commitments []common.Hash `json:"commitments"`
 }
 
-func TestReferenceCommitments(t *testing.T) {
```

Comment:
The looping logic in this test is actually incorrect when the input data length is exactly a multiple of BlockSize.  Added a reference test for that case which showed this test was broken but the reader versions are correct.  The `absorbLeafInput` method this tests is now private and only the reader ones are public so safe to remove this test.